File: Java/Maths/Find largest number from array - input values.java
import java.util.Scanner;
class largestNumberFromArray_input {
    public static void main(String args[]){
        Scanner input = new Scanner(System.in);
        System.out.print("How many number should have the array: ");
        int size = input.nextInt();
        int[] array = new int[size];
        int largest = 0;
        for (int i = 0; i < array.length; i++) {
            System.out.print("Enter the number-"+(i+1)+": ");
            array[i] = input.nextInt();
            if (i == 0) {
                largest = array[i];
            } else if (array[i] > largest) {
                largest = array[i];
            }
        }
        System.out.println("\nLargest number is: " + largest);
    }
}

File: Python/Maths/Find largest number from array - input values.py
size = int(input("How many number should have the array: "))
array = [None] * size
largest = 0
for i in range(0, len(array)):
    array[i] = int(input("Enter the number-"+str(i+1)+": "))
    if i == 0:
        largest = array[i]
    elif array[i] > largest:
        largest = array[i]

print("\nLargest number is:", largest)